src/conf_functions.rs

```diff
@@ -78,8 +80,13 @@ impl<const N: usize> Configuration<N, f64> for RealConfiguration<N> {
         }
     }
 
-    fn random_genes(&self, _rng: &mut R) -> [f64; N] {
-        [1.1; N] // FIXME
+    fn random_genes(&self, rng: &mut R) -> [f64; N] {
```

Comment:
Shouldn't we extract it to another configurable function instead of hardcoding it?

Reply:
I don't think so